molecule-runtime/src/quartzCoreMain/kotlin/app/cash/molecule/DisplayLinkClock.kt

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2023 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.molecule
+
+import androidx.compose.runtime.BroadcastFrameClock
+import androidx.compose.runtime.MonotonicFrameClock
+import kotlinx.cinterop.ObjCAction
+import platform.Foundation.NSRunLoop
+import platform.Foundation.NSSelectorFromString
+import platform.QuartzCore.CADisplayLink
+
+public actual object DisplayLinkClock : MonotonicFrameClock {
+
+  @Suppress("unused") // This registers a DisplayLink listener.
+  private val displayLink: CADisplayLink = CADisplayLink.displayLinkWithTarget(
+    target = this,
+    selector = NSSelectorFromString(this::tickClock.name),
+  )
```

Comment:
I'm looking at this again months later and this doesn't need a variable. It can just be in an `init` block 🤦